File: initializr/src/main/resources/templates/Application.java
package ${packageName};

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

@Configuration
@ComponentScan
@EnableAutoConfiguration
public class ${applicationName} {

    public static void main(String[] args) {
        SpringApplication.run(${applicationName}.class, args);
    }
}

File: initializr/src/main/resources/templates/Application.kt
package ${packageName}

import org.springframework.boot.SpringApplication<% if (useSpringBootApplication) { %>
import org.springframework.boot.autoconfigure.SpringBootApplication<% } else { %>
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration<% } %>
<% if (useSpringBootApplication) { %>
@SpringBootApplication<% } else { %>
@Configuration
@ComponentScan
@EnableAutoConfiguration <% } %>
open class ${applicationName}

fun main(args: Array<String>) {
    SpringApplication.run(${applicationName}::class.java, *args)
}
